MovieApp/MovieApp/App/Features/MovieBrowser/Components/MovieSearchBar+Design.swift

```diff
@@ -0,0 +1,139 @@
+import UIKit
+import SnapKit
+
+extension MovieSearchBar: DesignProtocol {
+    
+    func buildViews() {
+        createViews()
+        styleViews()
+        defineLayoutForViews()
+    }
+    
+    func createViews() {
+        searchGrayFieldView = UIView()
+        addSubview(searchGrayFieldView)
+        
+        let image = ImageEnum.searchBarIcon.image
+        searchBarIconImageView = UIImageView(image: image)
+        searchGrayFieldView.addSubview(searchBarIconImageView)
+        
+        searchTextField = UITextField()
+        searchTextField.delegate = self
+        searchGrayFieldView.addSubview(searchTextField)
+        
+        cancelButton = UIButton()
+        cancelButton.addTarget(self, action: #selector(cancelPressed), for: .touchUpInside)
+        addSubview(cancelButton)
+        
+        deleteTextButton = UIButton()
+        deleteTextButton.addTarget(self, action: #selector(deleteTextFieldEntry), for: .touchUpInside)
+        searchGrayFieldView.addSubview(deleteTextButton)
+    }
+    
+    func styleViews() {
+        searchGrayFieldView.layer.cornerRadius = grayFieldCornerRadius
+        searchGrayFieldView.backgroundColor = .searchFieldGray
+        
+        searchBarIconImageView.contentMode = .scaleAspectFit
+        
+        searchTextField.attributedPlaceholder = NSAttributedString(
+            string: "Search",
+            attributes: [NSAttributedString.Key.foregroundColor: UIColor.searchPlaceholderColor,
+                         NSAttributedString.Key.font: UIFont(name: Fonts.proximaMedium, size: 16) ?? .systemFont(ofSize: 16)])
```

Comment:
For fonts, you can create UIFont extension where you'll have sth like
```
public static let heading1: UIFont = .regular(size: 60)

...

static func regular(size: CGFloat = 12) -> UIFont {
  UIFont(with: .regular, size: size)
}
```

Reply:
I [created](https://github.com/lihter/Movie-App/pull/24/files#diff-019a1bcedcb8e120ecbea6c4a01e205958bbfa9c7e0a20366e002a99374c42e1R3) an extension as advised.